DependencyInjection/Configuration.php

```diff
@@ -71,6 +72,7 @@ public function addKeyChainsConfiguration(ArrayNodeDefinition $rootNode): self
                             ->scalarNode('public_key')->isRequired()->end()
                             ->scalarNode('private_key')->end()
                             ->scalarNode('private_key_passphrase')->end()
+                            ->scalarNode('algorithm')->defaultValue(KeyInterface::ALG_RS256)->end()
```

Comment:
library and bundle may evolve at different paces, that's why I explicitely put the default alg. in the tree buillder to avoid sudden breaking changes (that will be btw quite hard to debug).